app.js

```diff
@@ -68,8 +68,24 @@ const bodyParser= require('body-parser');
 var app = express();
 //https://expressjs.com/en/advanced/best-practice-security.html
 //https://helmetjs.github.io/docs/
+const environment = process.env.ENVIRONMENT || 'production';
 const helmet = require('helmet');
-app.use(helmet());
+const defaultCspOptions = helmet.contentSecurityPolicy.getDefaultDirectives();
+delete defaultCspOptions["upgrade-insecure-requests"]
+
+
+if (environment !== 'production') {
```

Comment:
I would prefer a positive check, so if environment is development override the defaults. For all other environment values use the default. Can you also add details around this in the README

Reply:
Sure.